src/main/java/org/launchcode/liftoffrecipetracker/models/Recipe.java

```diff
@@ -18,13 +18,13 @@
 @Indexed
 public class Recipe extends AbstractRecommendations {
 	//properties
-	@NotBlank
+	@NotBlank(message = "")
```

Comment:
If I put spaces in and nothing else, it doesn't proceed but there is no message to explain why.  (Obviously this is true for all the fields that have the "" error message.)  Something to think about.  Perhaps the size message could reflect the size requirements, and the @NotBlank could say the specific is required.  That would eliminate repetitive messaging while still allowing for an error message that reflects the situation.

Reply:
Updated this so the message is on the "NotBlank" and indicates both the required and size messaging. Switched the empty message to size. Tested to include all spaces and messaging worked at that point